org.oasis.xdita/dtd/lw-common.mod

```diff
@@ -61,6 +51,14 @@
              outputclass  CDATA          #IMPLIED
              class CDATA "- topic/xref ">
 
+<!--                    LONG NAME: Other Metadata                  -->
+<!ELEMENT othermeta        EMPTY>
+<!ATTLIST othermeta
+             %localization;
+             name         CDATA          #REQUIRED
+             content      CDATA          #REQUIRED
+             class        CDATA "- map/othermeta ">            
```

Comment:
The class attribute should use `topic/othermeta` rather than `map/othermeta` -- see: https://github.com/oasis-tcs/dita/blob/DITA-2.0/doctypes/dtd/base/metaDecl.mod#L506

Reply:
I took the othermeta definition from the original map DTD, but in order not to have them twice, I put them into lw-common.mod. Do you see any issues in the context of a map, if we use the class attribute as you suggest?